internal/fformat/fformat_test.go

```diff
@@ -33,7 +33,7 @@ func BenchmarkSiegfried(b *testing.B) {
 		sf := fformat.NewSiegfriedEmbed()
 		b.ResetTimer()
 		for i := 0; i < b.N; i++ {
-			sf.Identify("fformat.go")
+			_, _ = sf.Identify("fformat.go")
```

Comment:
Now that I noticed, the entire fformat package could be cleaned out too.

Reply:
@jraddaoui right, good point. :)